lib/rubocop/cop/layout/space_before_brackets.rb

```diff
@@ -37,13 +37,20 @@ def offense_range(node, begin_pos)
             receiver_end_pos = node.receiver.source_range.end_pos
             selector_begin_pos = node.loc.selector.begin_pos
             return if receiver_end_pos >= selector_begin_pos
+            return if brackets_without_syntax_sugar?(node, receiver_end_pos, selector_begin_pos)
 
             range_between(receiver_end_pos, selector_begin_pos)
           elsif node.method?(:[]=)
             offense_range_for_assignment(node, begin_pos)
           end
         end
 
+        def brackets_without_syntax_sugar?(node, receiver_end_pos, selector_begin_pos)
```

Comment:
Does this mean something like this?

```suggestion
        def dot_before_brackets?(node, receiver_end_pos, selector_begin_pos)
```

Reply:
Exactly, that was a littie weired naming :)